client/app/services/services.js

```diff
@@ -1,7 +1,8 @@
 angular.module('boundless.services', [])
 
 	//services to fetch & make groups
-.factory('Groups', function ($http) {
+.factory('Groups', function($http, $location, $window) {
```

Comment:
why did we need to inject $location and $window? Are they being used in this factory?